bring/src/main/java/com/bobocode/petros/bring/context/domain/BeanScope.java

```diff
@@ -19,4 +19,12 @@ public enum BeanScope {
     public static String getScopeAsString(final BeanScope scope) {
         return scope.name();
     }
+
+    public static BeanScope getScopeFromString(final String scope) {
```

Comment:
First of all, this method implementation can be replaced like this:
`public static BeanScope getScopeFromString(final String scope) {
        return Enum.valueOf(BeanScope.class, scope);
 }`
Moreover, its a bad idea to return prototype, if scope != singleton, would be better to fast-fail with an exception, and Enum.valueOf() provides this feature.

Also, please add javadoc for this method.